### ISSUE TITLE
`$HOME/.docker/config.json` created as directory if it doesn't already exist

### ISSUE THREAD

**Issue by jackross** [technekes/nib#31]
Using the alias described in the README:

```sh
alias nib='
  docker run \
    -it \
    --rm \
    -v $(pwd):$(pwd) \
    -w $(pwd) \
    -v $HOME/.docker/config.json:/root/.docker/config.json:ro \
    -v /var/run/docker.sock:/var/run/docker.sock \
    -e "DOCKER_HOST_URL=$DOCKER_HOST" \
    technekes/nib'
```

`nib` will result in `$HOME/.docker/config.json` being created as a directory if it does not already exist.  Additionally, if `$HOME/.docker/config.json` does exist (as an empty, touched file, for example), additional errors will be produced.  Example:

```sh
$ nib shell app
root@14338a3d2d19:/usr/src/app# exit

$ rm ~/.docker/config.json
$ nib shell app
  WARNING: Error loading config file:$HOME/.docker/config.json - read $HOME/.docker/config.json: is a directory
root@0890e6ac21e9:/usr/src/app# exit

$ rm -r ~/.docker/config.json
$ touch ~/.docker/config.json
$ nib shell app
  WARNING: Error loading config file:$HOME/.docker/config.json - EOF
root@5ce81b4826db:/usr/src/app# exit

$ docker login
  Login with your Docker ID to push and pull images from Docker Hub. If you dont have a Docker ID, head over to https://hub.docker.com to create one.
  Username: jackross
  Password:
  Login Succeeded
$ nib shell app
root@60fd68318218:/usr/src/app# exit
```

Typically, `$HOME/.docker/config.json` won't exist UNLESS you have used `docker login`...

**Comment by johnallen3d** [comment]
This may be just dodging the issue but I don't believe we currently have any features that require access to private repos (previously the `technekes/nib` project was private). Should we just update the alias in [README.md](https://github.com/technekes/nib/blob/master/README.md) for the time being?

/cc @blankenshipz

**Comment by blankenshipz** [comment]
@johnallen3d I think it's an acceptable solution, if someone had a dependency on a private repo they could theoretically issue a `docker pull` outside of `nib`.  Another option would be to change the alias into a function that optionally includes the volume when the file exists.

**Comment by johnallen3d** [comment]
Resolved by #35
